app/src/main/java/com/example/fooding/fragments/SearchFragment.java

```diff
@@ -0,0 +1,143 @@
+//
+//        rvRecipes.setLayoutManager(new LinearLayoutManager(getContext()));
+//
+//        recipeSearchView.setOnQueryTextListener(new SearchView.OnQueryTextListener() {
+//            @Override
+//            public boolean onQueryTextSubmit(String query) {
+//                aFood.clear();
+//
+//                fetchFood(query);
+//
+//                recipeSearchView.clearFocus();
+//                return false;
+//            }
+//
+//            @Override
+//            public boolean onQueryTextChange(String newText) {
+//                return false;
+//            }
+//
+//            private void fetchFood(String query) {
+//                Log.i(TAG, "FETCHING Food");
+//                foodclient.getFood(query, new JsonHttpResponseHandler() {
+//                    @Override
+//                    public void onSuccess(int statusCode, Headers headers, JSON response) {
+//                        try {
+//                            JSONArray jsonResults;
+//                            if (response != null) {
+//                                // Get the jsonResults
+//                                jsonResults = response.jsonObject.getJSONArray("docs");
+//                                // Parse json array into array of model objects
+//                                // final ArrayList<Result> results = Result.fromJson(docs);
+//                                // add five books into the results array
+//                                int jsonResultsLength = jsonResults.length();
+//                                final ArrayList<Food> results;
+//                                if (jsonResultsLength > 0) {
+//                                    if (jsonResultsLength > 0)
+//                                        results = new ArrayList<>(Food.fromJson(jsonResults, "Food");
+//                                    else
+//                                        results = new ArrayList<>(Food.fromJson(jsonResults, "Food");
+//                                    // Load model objects into the adapter
+//                                    for (Food food : Foods) {
+//                                        aFood.add(food); // add result through the adapter
+//                                    }
+//                                    FoodAdapter.notifyDataSetChanged();
+//                                }
+//                            }
+//                        } catch (JSONException e) {
+//                            // Invalid JSON format, show appropriate error.
+//                            e.printStackTrace();
+//                        }
+//                    }
+//
+//                    @Override
+//                    public void onFailure(int statusCode, Headers headers, String response, Throwable throwable) {
+//                        Log.e(TAG,
+//                                "Request failed with code " + statusCode + ". Response message: " + response);
+//                    }
+//
+//
+//
+//
```

Comment:
as i've mentioned before if you commit comments here, please leave a TODO

Reply:
Noted